.coveragerc

```
@@ -0,0 +1,17 @@
+[run]
+source = repo_size_guardian
+branch = true
+
+[report]
+exclude_lines =
+    pragma: no cover
+    def __repr__
+    if self.debug:
+    if settings.DEBUG
+    raise AssertionError
+    raise NotImplementedError
+    if 0:
+    if __name__ == .__main__.:
```

```suggestion
    raise NotImplementedError
```
I'll add them when I needed.